packages/block-editor/src/store/actions.js

```diff
@@ -0,0 +1,540 @@
+/**
+ * External Dependencies
```

Comment:
This will be a lint error as of #13785 . You will need to rebase and/or resolve it.

There may be other instances in the changes proposed here.

Reply:
> This will be a lint error as of #13785 . You will need to rebase and/or resolve it.
> 
> There may be other instances in the changes proposed here.

This is resolved by the rebased 164bdd39d00958a84412e4e5db159c15137c098f.